### PR TITLE
[OCF-66] runtime/action-php-7: Switch to CentOS 7

### PR DESCRIPTION
**Pull request by goldmann**
https://issues.jboss.org/browse/OCF-66

Additionally the PHP image is now preloaded and a test resource was added.

### CODE REVIEW

File: docker/runtimes/action-php-7/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,34 @@
-FROM openwhisk/action-php-v7.1:1.0.0
+FROM jboss/base:latest
```
Comment (jcrossley3) [inline docker/runtimes/action-php-7/Dockerfile]: Nope!

File: persistent-template.yml [hunk 2]
```diff
@@ -912,6 +912,10 @@ objects:
           image: projectodd/action-python-2:${PROJECTODD_VERSION}
           imagePullPolicy: IfNotPresent
           command: ["/bin/sh", "-c", "echo", "success"]
+        - name: preload-openwhisk-php7
```
Comment (jcrossley3) [inline persistent-template.yml:915]: iirc, we hit some max on the number of init containers we could have here, but if you saw it work in a virgin minishift, i guess not anymore.

File: resources/vars.php [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?php
```
Comment (jcrossley3) [inline resources/vars.php:1]: Look at you adding a test! Very thorough! :)

File: template.yml [hunk 4]
```diff
@@ -892,6 +892,10 @@ objects:
           image: projectodd/action-python-2:${PROJECTODD_VERSION}
           imagePullPolicy: IfNotPresent
           command: ["/bin/sh", "-c", "echo", "success"]
+        - name: preload-openwhisk-php7
```
Comment (jcrossley3) [inline template.yml:895]: same comment as for persistent

**Review by jcrossley3 — CHANGES_REQUESTED** [review]
Something other than 'latest' please

**Comment by goldmann** [comment]
There is no other tag and I do not plan to add others to the `jboss/base` image ATM. If we redo the image we base on, we will end up with a different image to maintain which inherits exactly same centos image, which is a waste of time to do it and manage afterwards.

File: persistent-template.yml (hunk 2 above)
Comment (goldmann) [reply persistent-template.yml:915]: I haven't seen any immediate failures while recreating the project with this in the template. I assume that it should fail at this stage, so I think we're fine.

**Comment by goldmann** [comment]
@jcrossley3 This is now ready for another review.

**Review by jcrossley3 — APPROVED** [review]

**Pull request merged**